index.csv

```diff
@@ -205,6 +210,7 @@ source-map-loader,MIT,0.1.0,0.1.1,0.1.2,0.1.3,0.1.4,0.1.5,0.1.6,0.2.0,0.2.1,0.2.
 source-map-support,MIT,0.1.0,0.1.1,0.1.2,0.1.3,0.1.4,0.1.5,0.1.6,0.1.7,0.1.8,0.1.9,0.2.0,0.2.1,0.2.2,0.2.3,0.2.4,0.2.5,0.2.6,0.2.7,0.2.8,0.2.9,0.2.10,0.3.0,0.3.1,0.3.2
 ssh2,MIT,0.0.0,0.0.1,0.0.2,0.1.0,0.1.1,0.1.2,0.1.3,0.1.4,0.1.5,0.1.6,0.1.7,0.1.8,0.1.9,0.1.10,0.1.11,0.1.12,0.1.13,0.2.0,0.2.1,0.2.2,0.2.3,0.2.4,0.2.5,0.2.6,0.2.7,0.2.8,0.2.9,0.2.10,0.2.11,0.2.12,0.2.13,0.2.14,0.2.15,0.2.16,0.2.17,0.2.18,0.2.19,0.2.20,0.2.21,0.2.22,0.2.23,0.2.24,0.2.25,0.3.0,0.3.1,0.3.2,0.3.3,0.3.4,0.3.5,0.3.6,0.4.0,0.4.1,0.4.2,0.4.3,0.4.4,0.4.5,0.4.6,0.4.7,0.4.8,0.4.9,0.4.10,0.4.11,0.4.12,0.4.13,0.4.14,0.4.15,0.5.0,0.5.1,0.5.2,0.5.3,0.5.4,0.5.5,0.6.0,0.6.1,0.6.2,0.7.0,0.7.1,0.7.2,0.8.0,0.8.1,0.8.2
 string,MIT,0.0.1,1.0.0,1.1.0,1.2.0,1.2.1,1.3.0,1.3.1,1.4.0,1.5.0,1.5.1,1.6.0,1.6.1,1.7.0,1.8.0,1.8.1,1.9.0,1.9.1,2.0.0,2.0.1,2.1.0,2.2.0,3.0.0,3.0.1,3.1.0,3.1.1
+stubborn-fs,MIT,1.0.0,1.1.0,1.2.0,1.2.1,1.2.2,1.2.3,1.2.4,1.2.5
```

Comment:
Looks good. Author added license file in initial commit: https://github.com/fabiospampinato/stubborn-fs/commit/74e557be769a0f83b85e7150363d68c41406c5b7